extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -316,6 +313,112 @@ describes.realWin('amp-list component', {
     });
   });
 
+  describe('load more', () => {
```

Comment:
Another type of test to consider if these start to become too fragile/hacky, is integration tests using `describes.integration`. For those tests, essentially you provide some valid AMP HTML/CSS and they run as if it is a really AMP page, then you can use normal DOM API to wait for stuff and verify. So more blackbox testing closer to something you would normally do with WebDriver. Nothing can be mocked. 

If you search for `describes.integration` you can find some good examples. The `test-position-observer` one for example actually scrolls the page and then asserts stuff.